src/app/container/tool-file-editor/tool-file-editor.component.ts

```diff
@@ -113,7 +113,7 @@ export class ToolFileEditorComponent extends FileEditing {
           // Only stop editing when version change was successful (not 204)
           this.toggleEdit();
           this.containerService.setTool(editedDockstoreTool);
-          this.alertService.detailedSuccess();
+          this.alertService.detailedSuccess('Saved version ' + ' of hosted tool ');
```

Comment:
This looks like it's for another ticket. Which can be OK, but:

1. Please reference the ticket in your commit message
2. You should either mention the version name (preferred) or get rid of the unnecessary concatenation.

Reply:
I'm sorry, let me take this out